composer.json

```diff
@@ -30,5 +31,25 @@
       "type": "git",
       "url": "https://github.com/totten/paratest.git"
     }
-  ]
+  ],
+  "extra": {
+    "downloads": {
+      "amp": {"url": "https://download.civicrm.org/amp/amp.phar-2019-06-25-dc96b3da", "path": "bin/amp", "type": "phar"},
+      "box": {"url": "https://github.com/box-project/box2/releases/download/2.7.5/box-2.7.5.phar", "path": "bin/box", "type": "phar"},
+      "civici": {"url": "https://download.civicrm.org/civici/civici-0.1.3.phar", "path": "bin/civici", "type": "phar"},
```

Comment:
@totten just a small thing i notice that the composer is saying these are all essentially dev-master but that might be confusing when there is an actual version in the url? is there a version param we can specify also?

Reply:
That feels sensible to me right